ci/lint/format.sh

```diff
@@ -149,11 +149,13 @@ MYPY_FILES=(
 
 
 BLACK_EXCLUDES=(
-    '--force-exclude' 'python/ray/cloudpickle/*'
-    '--force-exclude' 'python/build/*'
-    '--force-exclude' 'python/ray/core/src/ray/gcs/*'
-    '--force-exclude' 'python/ray/thirdparty_files/*'
-    '--force-exclude' 'python/ray/_private/thirdparty/*'
+    '--force-exclude'
+    'python/ray/cloudpickle/*|'`
+    `'python/build/*|'`
+    `'python/ray/core/src/ray/gcs/*|'`
+    `'python/ray/thirdparty_files/*|'`
+    `'python/ray/_private/thirdparty/*|'`
+    `'python/ray/serve/tests/test_config_files/syntax_error\.py'
```

Comment:
I'm pretty sure `--force-exclude` only takes one regex string, so all the files need to be in that single regex string. When I tried adding `python/ray/serve/tests/test_config_files/syntax_error.py` as a `--force-exclude` [here](https://buildkite.com/ray-project/oss-ci-build-pr/builds/14136#0186c7a2-b24c-4a4c-9ca7-b4dc14560334), the linter ignored `syntax_error.py` but started reformatting a bunch of files in `python/ray/_private/thirdparty/*`, so the last `--force-exclude` probably overrides all the previous ones.